sections/image-banner.liquid

```diff
@@ -143,29 +143,26 @@
       "id": "image_height",
       "options": [
         {
-          "value": "small",
+          "value": "adapt",
           "label": "t:sections.image-banner.settings.image_height.options__1.label"
         },
         {
-          "value": "medium",
+          "value": "small",
           "label": "t:sections.image-banner.settings.image_height.options__2.label"
         },
         {
-          "value": "large",
+          "value": "medium",
           "label": "t:sections.image-banner.settings.image_height.options__3.label"
+        },
+        {
+          "value": "large",
+          "label": "t:sections.image-banner.settings.image_height.options__4.label"
         }
       ],
       "default": "medium",
       "label": "t:sections.image-banner.settings.image_height.label",
       "info": "t:sections.image-banner.settings.image_height.info"
     },
-    {
-      "type": "checkbox",
-      "id": "adapt_height_first_image",
-      "default": false,
-      "label": "t:sections.image-banner.settings.adapt_height_first_image.label",
-      "info": "t:sections.image-banner.settings.adapt_height_first_image.info"
```

Comment:
Should we remove these translation keys in `en.default.schema.json`?

Reply:
I actually did mean to initially. Do you know what happens if we do remove some? Do the generated strings end up getting removed from the other files async or anything or would we have to go and specifically delete them ourselves?

Reply:
I'm pretty sure it gets removed by the translation platform at some point if we remove it from the English file. But I'm okay with doing it on a separate PR.

Reply:
Hey team! Seems like the translation was not removed. TIL!
There is an open PR for removing it from our community https://github.com/Shopify/dawn/pull/2482